plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -696,6 +698,116 @@ void GSRendererDX11::ResetStates()
 	m_om_dssel.key = 0;
 }
 
+// Partial port of OGL SW blending. Currently only works for accumulation blend or when m_sw_blending
+// m_sw_blending >= ACC_BLEND_HIGH_D3D11 since it has not been fully tested. There is no check
+// currently for primitive overlap like in OGL but it should give better results for post-processing effects
+// that are done by non-overlapping sprites.
```

Comment:
Maybe move the comment inside the function instead, seems cleaner maybe.
Also maybe move the blend function below channel shuffle like on gl, kinda got used to search at the upper half.

Well these two are just minor nitpicks.